packages/transport/src/types.ts

```diff
@@ -51,30 +58,48 @@ export const isUnaryResponse = <S extends ServiceType>(
 export const unaryResponseMsg = <S extends ServiceType>(
   req: DappMessageRequest<S>,
   result: GrpcResponse<S>,
+  typeRegistry: IMessageTypeRegistry,
 ): UnaryResponse<S> => {
   return {
     type: OUTGOING_GRPC_MESSAGE,
     sequence: req.sequence,
-    requestTypeName: req.requestTypeName,
     serviceTypeName: req.serviceTypeName,
-    result,
+    requestTypeName: req.requestTypeName,
+    responseTypeName: result.getType().typeName,
+    result: result.toJson({ typeRegistry }),
```

Comment:
@turbocrime fyi

When attempting to send a protobuf `Any` to the frontend, it was throwing json serialization issues. Turns out, we weren't doing `.json()` ourselves and relying upon chrome's internal methods to do this. We didn't notice until now because our serialization requires a registry now.